src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php

```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
     $options += array('message' => FALSE);
 
     $entities = $this->entityManager->getStorage($this->getEntityTypeId())->loadMultiple($ids);
-    foreach ($entities as $node) {
-      \Drupal::service('pathauto.manager')->updateAlias($node, 'bulkupdate', $options);
+    foreach ($entities as $entity) {
+      \Drupal::service('pathauto.manager')->updateAlias($entity, 'bulkupdate', $options);
     }
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated 1 URL alias for type %label.', 'Updated @count URL aliases for type %label.'), array('%label' => $entity->getLabel()));
```

Comment:
$entity->getLabel() is e.g. a specific node title, $entity is the last entity from the loop above, that's not what you want. Instead, you need to get the entity type definition from the entity manager and then the label of that.